.goreleaser.yml

```
@@ -0,0 +1,132 @@
+    gcflags:
+      - all=-trimpath={{ dir .ArtifactPath | dir | dir | dir }}
+    ldflags:
+      - -X {{ .Env.REPO }}/internal/version.Version={{ if not .IsSnapshot }}v{{ end }}{{ .Version }}
+      - -X {{ .Env.REPO }}/internal/version.GitVersion={{ .Tag }}
+      - -X {{ .Env.REPO }}/internal/version.GitCommit={{ .FullCommit }}
+      - -X {{ .Env.REPO }}/internal/version.KubernetesVersion={{ .Env.K8S_VERSION }}
+    goarch:
+      - amd64
+      - arm64
+      - ppc64le
+      - s390x
+    goos:
+      - linux
+      - darwin
+    ignore:
+      - goos: darwin
+        goarch: arm64
+      - goos: darwin
+        goarch: ppc64le
+      - goos: darwin
+        goarch: s390x
+
+  # ansible-operator build steps
+  - id: ansible-operator
+    main: ./cmd/ansible-operator
+    binary: ansible-operator
+    env:
+      - CGO_ENABLED=0
+      - GO111MODULE=on
+      - REPO=github.com/operator-framework/operator-sdk
+    mod_timestamp: "{{ .CommitTimestamp }}"
+    asmflags:
+      - all=-trimpath={{ dir .ArtifactPath | dir | dir | dir }}
+    gcflags:
+      - all=-trimpath={{ dir .ArtifactPath | dir | dir | dir }}
+    ldflags:
+      - -X {{ .Env.REPO }}/internal/version.Version={{ if not .IsSnapshot }}v{{ end }}{{ .Version }}
+      - -X {{ .Env.REPO }}/internal/version.GitVersion={{ .Tag }}
+      - -X {{ .Env.REPO }}/internal/version.GitCommit={{ .FullCommit }}
+      - -X {{ .Env.REPO }}/internal/version.KubernetesVersion={{ .Env.K8S_VERSION }}
+    goarch:
+      - amd64
+      - arm64
+      - ppc64le
+      - s390x
+    goos:
+      - linux
+      - darwin
+    ignore:
+      - goos: darwin
+        goarch: arm64
+      - goos: darwin
+        goarch: ppc64le
+      - goos: darwin
+        goarch: s390x
+
+  # helm-operator build steps
+  - id: helm-operator
+    main: ./cmd/helm-operator
+    binary: helm-operator
+    env:
+      - CGO_ENABLED=0
+      - GO111MODULE=on
+      - REPO=github.com/operator-framework/operator-sdk
+    mod_timestamp: "{{ .CommitTimestamp }}"
+    asmflags:
+      - all=-trimpath={{ dir .ArtifactPath | dir | dir | dir }}
+    gcflags:
+      - all=-trimpath={{ dir .ArtifactPath | dir | dir | dir }}
+    ldflags:
+      - -X {{ .Env.REPO }}/internal/version.Version={{ if not .IsSnapshot }}v{{ end }}{{ .Version }}
+      - -X {{ .Env.REPO }}/internal/version.GitVersion={{ .Tag }}
+      - -X {{ .Env.REPO }}/internal/version.GitCommit={{ .FullCommit }}
+      - -X {{ .Env.REPO }}/internal/version.KubernetesVersion={{ .Env.K8S_VERSION }}
+    goarch:
+      - amd64
+      - arm64
+      - ppc64le
+      - s390x
+    goos:
+      - linux
+      - darwin
+    ignore:
+      - goos: darwin
+        goarch: arm64
+      - goos: darwin
+        goarch: ppc64le
+      - goos: darwin
+        goarch: s390x
+
+# Use most recent tag and short commit for snapshot version.
+snapshot:
+  name_template: "{{ .Tag }}"
+
+# We don't use archives, so skip creating them.
+archives:
+  - format: binary
```

I think we may need the following to make the binaries have the same naming scheme we've been using:
```suggestion
  - format: binary
    replacements:
      amd64: x86_64
      darwin: apple-darwin
      linux: linux-gnu
      arm64: aarch64
    name_template: '{{.Binary}}-{{.Env.GIT_VERSION}}-{{.Arch}}-{{.Os}}'
```

---

One caveat: I noticed that our arm64 binary release on github use `aarch64` and our images in quay use `arm64`. They have the same meaning, just a different name. But I think at some point we might want to align those if that's possible as a non-breaking change.

---

Is it a breaking change to modify the released binary names?

---

Also removing version from binary names removes one more doc we have to change in the release commit (there are other ways around the latter issue, but this is the simplest).